### ISSUE TITLE
Potential bug in include headers

### ISSUE THREAD

**Issue by azplanlos** [jmcnamara/libxlsxwriter#71]
Hi,

I think there is a bug in the projects include header file include/xlsxwriter/common.h:

Two files are referenced by using a relative path that doesn't exist. Xcode is complaining about this when adding the library to my own framework.

```
#include "xlsxwriter/third_party/queue.h"
#include "xlsxwriter/third_party/tree.h"
```

Corrected like this it is compiling and working.

```
#include "third_party/queue.h"
#include "third_party/tree.h"
```

If this is really a bug, please correct it or let me know so I can open a pull request.

Andreas

**Comment by jmcnamara** [comment]
Hi,

Thanks for the report. I'll have a look at it.

John

**Comment by jmcnamara** [comment]
Fixed in version 0.5.6.

Thanks for the report.
